model_analyzer/record/metrics_manager.py

```diff
@@ -563,16 +563,17 @@ def _print_run_config_info(self, run_config):
         collect_cpu_metrics_expect = cpu_only or len(self._gpus) == 0
         collect_cpu_metrics_actual = len(self._cpu_metrics) > 0
         if collect_cpu_metrics_expect and not collect_cpu_metrics_actual:
-            logger.info(
-                "CPU metric(s) are not being collected, while this profiling will run on CPU(s)."
-            )
+            if not self._cpu_warning_printed:
+                self._cpu_warning_printed = True
+                logger.warning(
+                    "One or more models are running on the CPU, but CPU metric(s) are not being collected"
+                )
         # Warn user about CPU monitor performance issue
         if collect_cpu_metrics_actual:
             if not self._cpu_warning_printed:
                 self._cpu_warning_printed = True
-                logger.warning("CPU metric(s) are being collected.")
                 logger.warning(
-                    "Collecting CPU metric(s) can affect the latency or throughput numbers reported by perf analyzer."
+                    "CPU metrics are being collected. This can affect the latency or throughput numbers reported by perf analyzer."
```

Comment:
Can we make it multiple lines for better readability?

Reply:
yapf/format.py should be taking care of this for us. I'm looking into it.